src/pages/Defi/components/OpportunityCardList.tsx

```diff
@@ -39,7 +56,13 @@ export const OpportunityCardList = ({ balances }: { balances: UseEarnBalancesRet
         gridGap={6}
       >
         {activeOpportunities.map(opportunity => {
-          return <OpportunityCard key={opportunity.assetId} {...opportunity} />
+          return (
+            <OpportunityCard
+              key={opportunity.assetId}
+              {...opportunity}
+              overrideAssetIconID={overrideIconAssetId}
```

Comment:
why does this have two different names? `overrideAssetIconID` doesn't make sense, capitalization isn't consistent.